android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
and again  another gradle wrapper for android? same question as for `buildSrc` - can we use the one from root project?

Reply:
Well no for 2 reasons
- root project uses Gradle 6 and kotlin android does not like Gradle 6
- the android project is separate from the main project that way we get around intellij turning everything into an Android project